eos-updater-flatpak-installer/eos-updater-flatpak-installer-fallback.timer.in

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
A bit pointless for this to be a `.in` file that's configured right? Since it's already complete. Though that criticism also applies to `eos-autoupdater.timer.in`.

Reply:
Keeps the `meson.build` simple